<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kbvHelpInfoClass</class>
 <widget class="QDialog" name="kbvHelpInfoClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <property name="topMargin">
     <number>4</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QTabWidget" name="infoTabs">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string notr="true"/>
      </property>
      <property name="statusTip">
       <string notr="true"/>
      </property>
      <property name="whatsThis">
       <string notr="true"/>
      </property>
      <property name="accessibleName">
       <string notr="true"/>
      </property>
      <property name="accessibleDescription">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabInfo">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="statusTip">
        <string notr="true"/>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <property name="accessibleName">
        <string notr="true"/>
       </property>
       <property name="accessibleDescription">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string notr="true">&amp;Info</string>
       </attribute>
       <widget class="QLabel" name="infoCopyrightLbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>240</y>
          <width>421</width>
          <height>21</height>
         </rect>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">copyright info</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="infoIconLbl">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>101</width>
          <height>101</height>
         </rect>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="pixmap">
         <pixmap resource="../res/kbv.qrc">:/kbv/icons/imarca.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="infoVersionLbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>421</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>26</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">Imarca  0.0.0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="infoAuthorLbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>421</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="infoBuildLbl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>421</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">Build date</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabLicence">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string notr="true"/>
       </property>
       <property name="statusTip">
        <string notr="true"/>
       </property>
       <property name="whatsThis">
        <string notr="true"/>
       </property>
       <attribute name="title">
        <string notr="true">&amp;Licence</string>
       </attribute>
       <widget class="QPlainTextEdit" name="licenceTextEdit">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>441</width>
          <height>321</height>
         </rect>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="overwriteMode">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
        <property name="placeholderText">
         <string notr="true"/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabPlugins">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>&amp;Plugins</string>
       </attribute>
       <widget class="QTableWidget" name="pluginsTable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>444</width>
          <height>320</height>
         </rect>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <property name="columnCount">
         <number>4</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>24</number>
        </attribute>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
      </widget>
      <widget class="QWidget" name="tabSpecial">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Special</string>
       </attribute>
       <widget class="QLabel" name="specialLblDescription">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>411</width>
          <height>131</height>
         </rect>
        </property>
        <property name="toolTip">
         <string notr="true"/>
        </property>
        <property name="statusTip">
         <string notr="true"/>
        </property>
        <property name="whatsThis">
         <string notr="true"/>
        </property>
        <property name="accessibleName">
         <string notr="true"/>
        </property>
        <property name="accessibleDescription">
         <string notr="true"/>
        </property>
        <property name="text">
         <string notr="true">Upgrade  database from Imarca version 0.3.36 to version 0.3.37 where all file paths are local paths below the collection root path.</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="specialCombo">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>221</width>
          <height>23</height>
         </rect>
        </property>
        <property name="currentText">
         <string notr="true"/>
        </property>
       </widget>
       <widget class="QPushButton" name="specialButtonApply">
        <property name="geometry">
         <rect>
          <x>310</x>
          <y>230</y>
          <width>80</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="specialProgress">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>160</y>
          <width>51</width>
          <height>23</height>
         </rect>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="infoButtClose">
      <property name="minimumSize">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
      <property name="centerButtons">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../res/kbv.qrc"/>
 </resources>
 <connections/>
</ui>
